pages/community.html

```diff
@@ -27,13 +31,12 @@
 The following channels also have a strong presence of Coq users:
 </p>
 <ul>
-<li><a href="https://stackoverflow.com/">Stack Overflow</a> (use the
-<a href="https://stackoverflow.com/questions/tagged/coq">coq</a> tag).
-<li>
-<a href="https://cstheory.stackexchange.com">TCS Stack Exchange</a>
-(which also has
-a <a href="https://cstheory.stackexchange.com/questions/tagged/coq">coq</a>
-tag) for questions on the meta-theory of Coq.
+<li>Other sites in the Stack Exchange galaxy, such
```

Comment:
```suggestion
<li>Other sites in the Stack Exchange Q&amp;A site galaxy, such
```
If we say Q&A in the first place, I think it makes sense to repeat.